sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Diagnostics/BlobCheckpointStoreInternal.Diagnostics.cs

```diff
@@ -291,7 +291,7 @@ partial void GetCheckpointComplete(string fullyQualifiedNamespace,
                                            string partitionId,
                                            string clientIdentifier,
                                            DateTimeOffset lastModified) =>
-            Logger.GetCheckpointComplete(fullyQualifiedNamespace, eventHubName, consumerGroup, partitionId, clientIdentifier, lastModified);
+            Logger.GetCheckpointComplete(fullyQualifiedNamespace, eventHubName, consumerGroup, partitionId, clientIdentifier, lastModified.ToString());
```

Comment:
We probably want to set an explicit format specifier for the conversion here.   For example, we use this [elsewhere](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs/src/Diagnostics/EventHubsEventSource.cs#L2202):

```csharp
lastModified.Value.ToString("yyyy-mm-ddTHH:mm:ss.fffZ", CultureInfo.InvariantCulture)
```

Reply:
Do you copy that string around in the messaging code?

Reply:
We log that string as part of our event source.